admin_guide/metrics-sizing-guidelines.adoc

```diff
@@ -0,0 +1,164 @@
+= Openshift Metrics Sizing Guidelines
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Openshift Metrics Basics
+
+Monitoring openshift pods is important task from both, Openshift administrator and Openshift user point
```

Comment:
is an important task from both the OpenShift adminstrator and users point of view.